.github/workflows/flutter.yaml

```
@@ -0,0 +1,36 @@
+          channel: 'stable'
+
+      - run: flutter pub get
+      - run: flutter analyze
+
+      - name: Build Linux
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ninja-build libgtk-3-dev
+          flutter build linux --release
+      - name: Upload Linux Artifact
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)